src/read_from_file/handle_battle_cmd.cpp

```diff
@@ -19,6 +19,16 @@ static void ExtractCreatureFromFile(std::string& foe_name, std::vector<Creature>
 	std::unordered_map<std::string, std::string> foe_stats_map =
 		utils::ReadFromJSON(file_path);
 
+	std::string foe_display_name = foe_name;
+
+	for (size_t i = 0; i < foe_display_name.length(); i++)
```

Comment:
That's nice. However, I would propose to extract this one into a utils function (see also similar comment for capitalizing first letter).